js-envs/src/main/scala/org/scalajs/jsenv/phantomjs/PhantomJSEnv.scala

```diff
@@ -230,6 +228,13 @@ class PhantomJSEnv(
       new MemVirtualJSFile("comSetup.js").withContent(code)
     }
 
+    override def start(logger: Logger, console: JSConsole): Future[Unit] = {
+      setupLoggerAndConsole(logger, console)
+      mgr.start()
```

Comment:
I take it the `mgr` doesn't like to `start()` before the `logger` and `console` are ready?